collectors/python.d.plugin/dnsdist/dnsdist.chart.py

```diff
@@ -105,7 +105,7 @@
         ]
     },
     'query_latency_avg': {
-        'options': [None, 'Average latency for the last N queries', 'ms/query', 'latency',
+        'options': [None, 'Average latency for the last N queries', 'us/query', 'latency',
```

Comment:
i think just `microsecond` or `us` will do, lets remove `/query` suffix. What do you think?

Reply:
I like `microseconds` or even `useconds`. The `/query` probably isn't necessary, and just `us` could be confusing.

Reply:
lest make `microseconds`

Reply:
Done. Thank you!